### DISCUSSION TITLE
Bang Gini bang jadi gw ada di sesi 4 Next Js CUY UNIVERSE S4, nah ini saya udah ikutin tapi kayak aneh bet nih bang

### DISCUSSION THREAD

**Discussion by RASZZZKING** [deaafrizal/forum#61] [Troubleshooting Room] [answered]
Nah jadi Keanehannya itu adalah ketika gua memasukan yang namanya data page dari Usestate, nah ketika gua muculkan untuk 1 detik pertama itu muncul nilai nya bang, tapi ketika 2 detik abis refresh gitu jadi undefiend bang nilainya. kalau di masukan ke console log aman aja bang ada nilainya

gambar 1detik setelah refresh :
![image](https://github.com/deaaprizal/forum/assets/155736103/caeb78f6-9591-4cd6-aef5-3423a43c0532)

gambar setlah 2detik refresh :
![image](https://github.com/deaaprizal/forum/assets/155736103/ca084813-01b4-467a-b1da-b6c38a13344a)

ini struktur folder ku:
![image](https://github.com/deaaprizal/forum/assets/155736103/a2586cfb-247c-4c0d-85d9-97e924a78b8c)

udah usaha buat nyari sampai bikin ulang, tetep ngak nemu bang. mohon bantuannya biar ngak stuck terus bang,

**Comment by RASZZZKING** [comment, marked as answer]
udah nemu gais
![image](https://github.com/deaaprizal/forum/assets/155736103/15b7ff50-8e2f-4cfd-9070-87369a369601)

disini salahnya kelebihan kurung kurawal

**Reply by deaafrizal** [reply]
ahahah mantep2, udah panik ya kaaaann
